### ISSUE TITLE
undefined type User in src/Canvas.php

### ISSUE THREAD

**Issue by fpolli** [austintoddj/canvas#1038]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Fresh install of Canvas per the instructions here, and I notice that Visual Studio Code throws three errors of the type in the title. The lines are here:

`
    /**
     * Return an array of available user roles.
     *
     * @return array
     */
    public static function availableRoles(): array
    {
        return [
            User::CONTRIBUTOR => 'Contributor',
            User::EDITOR => 'Editor',
            User::ADMIN => 'Admin',
        ];
    }
`
I noticed that nowhere in the class does it say "use User", so I thought, "hey, maybe try adding that," and VSC just threw a fourth error, so clearly that was not the problem.

Thank you, and I will provide any more information needed.

### Expected Behavior

I expected to see no errors. :)

### Steps To Reproduce

I'm using Laravel 9/Jetstream in a Docker container made with Sail

### Environment

```markdown
I believe the Docker container is using PHP 8.1.

The latest Laravel as of a couple days ago, 9.?
```

### Anything else?

Further details upon request.

Best wishes for a great day.

Much appreciation for all the work in this.

**Comment by fpolli** [comment]
So I think this might be an issue with the Intelephense extension to VSC rather than Canvas. I was working in VSC again, and I noticed that the Canvas.php file throws no errors when I open it in VSC. I didn't do anything. I opened the PostController.php file, and it supposedly has 11 errors. It did not flag errors before (my VSC shows the filename in pink with a pink dot next to it if it thinks there are errors in the file). The errors? Undefined type. I am attaching a screenshot of the errors
<img width="653" alt="Screen Shot 2022-04-09 at 5 00 41 AM" src="https://user-images.githubusercontent.com/5469054/162573518-ffeac455-42e4-46bc-a628-463c4b0bcae6.png">

I'm a little new to a lot of this, so I apologize if I am wasting your time.
